frontend/src/components/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { FormControl, TextField, Button, Typography } from '@mui/material';
+
+import {
+  TextFieldWrap,
+  WrapH1,
+  BorderForm,
+  RegistrationFormWrapper,
+  StyledSpan,
+  WrapButtonAndText
+} from './styles';
+
+function ForgotPassword() {
+  return (
+    <RegistrationFormWrapper>
+      <BorderForm>
+        <FormControl sx={{ width: '35ch' }}>
+          <WrapH1>
+            <Typography>Forgot password?</Typography>
+          </WrapH1>
+
+          <TextFieldWrap>

```

Comment:
your own styled textfield blocks label in original textfield

therefore it's not displayed

Reply:
done